index.html

```diff
@@ -1,9 +1,9 @@
 ---
 ---
-{% assign dataVariable = site.playbookData["en"] %}{% 
-assign dataSourceEn = site.data[dataVariable] %}{% 
-assign dataVariable = site.playbookData["fr"] %}{% 
-assign dataSourceFr = site.data[dataVariable] %}
+{% assign dataVariable = site.playbookData["en"] %}
+{% assign dataSourceEn = site.data[dataVariable] %}
+{% assign dataVariable = site.playbookData["fr"] %}
+{% assign dataSourceFr = site.data[dataVariable] %}

```

Comment:
The reason that I had %}{% was to prevent the generation of blank lines as the new line between the %} on the one line and the {% on the next line was being output to the rendered page. This is consistent with most other files. The blank lines aren't really a big deal in this case, since it is an HTML file, but it makes a huge difference with markdown files.

Reply:
Originally the IDE had trimmed the trailing whitespace from those lines and I thought if there was going to be a diff, then this would be better. I can see if the trimmed trailing space version would work, but I thought that was more likely to cause a compiling issue

Reply:
The trimmed trailing space seemed to work fine in other files so should be okay here as well (since newline is treated as whitespace by Jekyll/Liquid).